dockerfiles/Dockerfile.android-ci-base

```diff
@@ -30,7 +27,7 @@ COPY build.gradle settings.gradle /app/
 
 # Build the example app - needed for Espresso tests
 COPY examples/ examples/
-RUN ./gradlew -p examples/sdk-app-example assembleRelease
+RUN cd examples/sdk-app-example && ./gradlew assembleRelease
```

Comment:
This is a better test that the example app builds - if something was wrong with the example app's gradle wrapper the old command would not have caught this.